app/models/tenant.rb

```diff
@@ -175,6 +175,10 @@ def editable_domains
     ae_domains.where(:system => false).order('priority DESC')
   end
 
+  def self.current_tenant
+    User.current_user.try(:current_tenant) || Tenant.default_tenant
```

Comment:
@kbrock how can i get current tenant of a logged in user, let me know if there is another way to do it, i can change code to use that.